community/elasticsearch/APKBUILD

```diff
@@ -19,6 +20,7 @@ source="https://artifacts.elastic.co/downloads/$pkgname/$pkgname-$pkgver.tar.gz
 builddir="$srcdir/$pkgname-$pkgver"
 
 default_module=transport-netty4
+mkdir -p $builddir/modules
 _modules=$(find $builddir/modules -mindepth 1 -maxdepth 1 -type d ! -name $default_module -exec basename {} \;)
```

Comment:
the `find` worked & created the subpkgs correctly but I've removed it. Now the module names are added manually as they were before. When the package is updated new modules will need to be checked for manually.

the `initd` was cleaned up because in `ES` now options for `java` are set in `jvm.options` (previously they were not). I thought it was more intuitve to set options for `java` in the one place they are expected to be (& not also have them set in the `initd`)

I've been running `ES` for over a week with `supervise-daemon` with no problems.